composer.json

```diff
@@ -136,7 +136,8 @@
         "symfony/doctrine-bridge":      "^6.0",
         "symfony/error-handler":        "^6.0",
         "symfony/http-foundation":      "^6.0",
-        "symfony/password-hasher":      "^6.0",
+        "symfony/http-kernel":          "^6.0",
+	"symfony/password-hasher":      "^6.0",
```

Comment:
You need to use spaces instead of tabs

Reply:
Thanks for pointing that out, I didn't even notice I changed that.